### ISSUE TITLE
Can I pick some processes and do "disable hook"?

### ISSUE THREAD

**Issue by arcticPeng93** [mgth/LittleBigMouse#277]
I watched your commit `cd7e7cf` and noticed there is an "Excluded list file" exsist.
Can I edit that txt file by my self?

or can you just disable hook "destiny2.exe" process please? I have a mouse stutter issue now. :)

**Comment by mgth** [comment]
Yes you can add a line to c:\ProgramData\Mgth\LittleBigMouse\Excluded.txt

You want  to add a signifiant part of the file path or the filename itself, just ensure the it will not match another program.

Note that for now you have to quit and restart LBM in order to have it loaded.

Also, please anyone finding a soft/game that should be excluded post here.
